src/openms/include/OpenMS/FILTERING/DATAREDUCTION/IsotopeDistributionCache.h

```diff
@@ -36,27 +36,50 @@
 
 #include <OpenMS/KERNEL/StandardTypes.h>
 #include <OpenMS/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderAlgorithmPickedHelperStructs.h>
+#include <OpenMS/CHEMISTRY/ISOTOPEDISTRIBUTION/IsotopeDistribution.h>
 
 namespace OpenMS
 {
   /**
    * @brief Pre-calculate isotope distributions for interesting mass ranges
    */
-  class OPENMS_DLLAPI IsotopeDistributionCache
+  class OPENMS_DLLAPI IsotopeDistributionCache:
+          public FeatureFinderAlgorithmPickedHelperStructs
```

Comment:
why inherit from `FeatureFinderAlgorithmPickedHelperStructs`????

Reply:
sorry, I forget to delete it.